src/app.js

```diff
@@ -1 +1,27 @@
+/* eslint-disable no-console */
 'use strict';
+
+const [operation, ...args] = process.argv.slice(2);
+
+const argsFilter = args.filter(num => !isNaN(Number(num)));
+
+if (argsFilter.length === 0) {
+  console.log(`There is nothing to ${operation}`);
+}
+
+let operationResult;
+
+switch (operation) {
+  case 'add':
+    operationResult = argsFilter.reduce((a, b) => Number(a) + Number(b), 0);
+    break;
+
+  case 'multiply':
+    operationResult = argsFilter.reduce((a, b) => Number(a) * Number(b), 1);
```

Comment:
I think it's better to convert to number using map method, so you can make your code more short and readable